prow/gerrit/adapter/adapter.go

```diff
@@ -397,7 +397,8 @@ func (c *Controller) handleInRepoConfigError(err error, instance string, change
 	if err != nil {
 		// If we have not already recorded this failure send an error essage
 		if failed, ok := c.inRepoConfigFailures[key]; !ok || !failed {
-			if setReviewWerr := c.gc.SetReview(instance, change.ID, change.CurrentRevision, inRepoConfigFailed, nil); setReviewWerr != nil {
+			msg := fmt.Sprintf("%v: %s", err, inRepoConfigFailed)
```

Comment:
nit: For readability I'd expect the constant message about this being an inrepo config error to come before the specific error details so that the reader has context.

I think we should also consider updating the wording of `inRepoConfigFailed` to mention that the files in the `.prow` directory or `.prow.yaml` may be improperly formatted.

Reply:
done